ops/fhir_validations/validations_ignorelist.yml

```diff
@@ -0,0 +1,51 @@
+ignore_list:
+  global_filter:
+    error_patterns:
+      - ".*Unknown Code System '.*/C4BBIdentifierType'"                   # Unfixable (6 errors)
+      - ".*is not valid in the system http://hl7.org/fhir/sid/icd-10.*"   # Invalid proc code for system (112 errors)
+      - ".*Unknown Code.*"                                                # Invalid code (3 errors)
+      - ".*None of the codings provided are in the value set.*"           # Invalid diag code for system (244 errors)
+      - ".*IGNORED_FIELD.*"                                               # Illegal value (142 errors)
+      - ".*must have a fullUrl which is the identity.*"                   # Missing bundle entry property (28 errors)
+      - ".*missing a fullUrl*"                                            # Missing bundle entry property (39 errors)
+      - ".*entry missing fullUrl*"                                        # Missing bundle entry property (22 errors)
+      - ".*Bundle.id.*Invalid Resource id.*"                              # Invalid bundle id (7 errors)
+      - ".*The extension .* is unknown, and not allowed here.*"           # Unresolvable url (1497 errors)
+      - ".*URL value 'http://hl7.org/fhir/sid/us-mbi' does not resolve.*" # Unresolvable URL (23 errors)
+      - ".*URL value 'http://hl7.org/fhir/sid/us-medicare' does not resolve" # Unresolvable URL (9 errors)
+      - ".*Coding\\.system must be an absolute reference, not a local reference.*" # Coding System Reference (6 errors)
+  file_filter:
+    - file_pattern: .*/claim.*\.json
+      error_patterns:
+        - ".*Claim\\.insurance: minimum required = 1, but only found 0.*"             # Missing required field (2 errors)
```

Comment:
@KyRobbins I believe we should be able to remove these 2 patterns in #972, when this merges could you remember to update this file in that branch.